builtin/logical/ssh/path_config_ca.go

```diff
@@ -228,6 +228,17 @@ func (b *backend) pathConfigCAUpdate(req *logical.Request, data *framework.Field
 
 		return nil, err
 	}
+
+	if generateSigningKey {
+		response := &logical.Response{
+			Data: map[string]interface{}{
```

Comment:
I think this is a useful enhancement, and I also think adding a read endpoint to the config/ca path that returns the public key as a normal API object would be good too.